dataverse-webapp/src/main/webapp/filesFragment.xhtml

```diff
@@ -117,6 +117,29 @@
                             <span class="glyphicon glyphicon-download-alt"/> <h:outputText value=" #{bundle['file.rsyncUpload.step2.downloadScriptButton']}"/>
                         </h:commandLink>
 
+                        <!-- DOWNLOAD ALL FILES BUTTON -->
+                        <p:outputPanel style="display: inline-block"
+                             rendered="#{!(empty datasetFilesTab.workingVersion.fileMetadatas)
+                                        and datasetFilesTab.downloadButtonAvailable}">
+                            <button type="button" class="btn btn-default btn-access dropdown-toggle" data-toggle="dropdown">
+                                <span class="glyphicon glyphicon-download-alt"/> #{bundle['file.download.dataset']} <span class="caret"/>
+                            </button>
+                            <ul class="dropdown-menu multi-level text-left" style="left:auto" role="menu">
+                                <li>
+                                    <p:commandLink action="#{datasetFilesTab.startDatasetFilesDownload(true)}"
+                                                   id="downloadOriginalFiles">
+                                        <h:outputText value="#{bundle['file.download.dataset.original']}"/>
+                                    </p:commandLink>
+                                </li>
+                                <li>
+                                    <p:commandLink action="#{datasetFilesTab.startDatasetFilesDownload(false)}"
+                                                   id="downloadFormated">
+                                        <h:outputText value="#{bundle['file.download.dataset.tab']}"/>
+                                    </p:commandLink>
+                                </li>
+                            </ul>
+                        </p:outputPanel>
```

Comment:
Why we don't do this like in downloading selected? That is one button if there is no tabular file and expandable button with 2 options if there is tabular file